devices/wholeBodyDynamics/SixAxisForceTorqueMeasureHelpers.cpp

```diff
@@ -9,19 +10,32 @@ SixAxisForceTorqueMeasureProcessor::SixAxisForceTorqueMeasureProcessor()
     // Initialize the affice function to be the identity
     toEigen(m_secondaryCalibrationMatrix).setIdentity();
     m_offset.zero();
+    toEigen(m_temperatureCoefficients).setZero();
+    m_tempOffset=0.0;
+    m_estimated_offset.zero();
```

Comment:
@traversaro If you call calib all it should not overwrite the offset obtained from the configuration file since it is stored separately. At least that was the intention. This way it kept backward compatibility and the preestimated offset is not lost in case the user inadvertently used calib all.

@HosameldinMohamed  you could verify if its the case.

Reply:
`m_estimated_offset` keeps the offset from the configuration file while the computed offset by calib all is always stored in `m_offset`. So calib all will overwrite temporarily the offest from the configuration file but it can be restored by calling the `usePreEstimated` offset function from rpc

Reply:
Thanks @fjandrad, I missed that detail.

Reply:
Not entirely useless. You can alternate between one and the other calling `calib all `and `usePredefinedOffset` from rpc at any time.

Reply:
Yes, totally. I was actually referring to the `calib all` that from this comment https://github.com/robotology/whole-body-estimators/pull/45#issuecomment-659975787 I understood that @HosameldinMohamed is using for his tests.

Reply:
> @traversaro If you call calib all it should not overwrite the offset obtained from the configuration file since it is stored separately. At least that was the intention. This way it kept backward compatibility and the preestimated offset is not lost in case the user inadvertently used calib all.
> 
> @HosameldinMohamed you could verify if its the case.

We couldn't do it this time. We will try to test this feature next time.

Reply:
Yes we noticed that.